src/client/menu/menu.c

```diff
@@ -1559,6 +1559,11 @@ static const char *idcredits[] = {
 	"Marty Stratton",
 	"Henk Hartong",
 	"",
+	"+Patches authors:"
+	"PGM",
+	"PMM",
```

Comment:
Do we really need to add PGM and PMM here? That are Peter Mack and Patrick Magruder, already credited in the Ground Zero sections. That just did some changes to the client and the renderes to add support for various new effects used in Ground Zero, mostly the weapons and the new power ups.

Reply:
@Yamagi I tried to save authors of patches, and if they are in list already, i have removed it from changes.